stdlib/funcs_bench_test.go

```diff
@@ -16,7 +16,7 @@ import (
 
 func BenchmarkTmAllTrueLiteralList(b *testing.B) {
 	b.StopTimer()
-	evalctx := eval.NewContext(stdlib.Functions(test.TempDir(b)))
+	evalctx := eval.NewContext(stdlib.Functions(test.TempDir(b), nil))
```

Comment:
Why is this nil and in all other cases its an empty list?

Reply:
good catch! I switched from `nil` to empty list because it's more clear that the intention is "no experiments". I forgot to update here.
I will merge and address this in another PR.
Thanks.